src/main/resources/swagger-ui/index.html

```diff
@@ -71,12 +71,12 @@
 <script src="./swagger-ui-standalone-preset.js"> </script>
 <script>
 window.onload = function() {
-
+  
   // Build a system
   const ui = SwaggerUIBundle({
-    spec: /*intellij-swagger-spec-start*//*intellij-swagger-spec-end*/,
```

Comment:
We need the `spec` and `validatorUrl`, the spec is where the plugin will inject the JSON Open API specification.